### ISSUE TITLE
Unused parameter reported for overload signatures

### ISSUE THREAD

**Issue by stephanedr** [palantir/tslint#1405]
### Bug Report
- **TSLint version**: `3.13.0`
- **TypeScript version**: `1.8.10`
- **Running TSLint via**: vscode
#### TypeScript code being linted

``` ts
function test(a: number): number;  // Unused parameter 'a'
function test(a: string): string;  // Unused parameter 'a'
function test(a: number | string): number | string { return a; }
```

with `tslint.json` configuration:

``` json
        "no-unused-variable": [ true, "check-parameters" ]
```
#### Actual behavior

See reported errors in given code.
#### Expected behavior

No errors for overload signatures (but still checked for the implementation).

**Comment by andy-hanson** [comment]
This seems to be fixed now.
